CONTRIBUTING.rst

```diff
@@ -0,0 +1,12 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+Before contributing code, please note:
+
+* You agree to license your contributions under the [license](https://github.com/mozilla/kinto/blob/master/LICENSE).
+* Please read the [contributing documentation](http://kinto.readthedocs.org/en/latest/contributing.html).
```

Comment:
We could quickly extract the most important things from this doc, put them here and add a link to "see you to contribute in more details"?